src/Routes/AppRoute.js

```diff
@@ -10,6 +11,8 @@ function AppRoute() {
         <Route path="/Home" element={<Home />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
+        <Route path="/expenseinput" element={<ExpenseInformation />} />
+        <Route path="/expensemonth" element={<ExpenseMonth />} />
```

Comment:
change path to /expense-input or /expense/input
same for month

Reply:
Done!